interactive_ai/supported_models/geti_supported_models/manifests/instance_segmentation/base.yaml

```diff
@@ -0,0 +1,12 @@
+task: instance_segmentation
+capabilities:
+  tiling: true
+
+hyperparameters:
+  evaluation:
+    metric: f_measure
```

Comment:
Segmentation uses "dice", doesn't it?

Reply:
I think both use dice? CC @maxxgx

Reply:
[Done](https://github.com/open-edge-platform/geti/pull/547/commits/625e8e7fb3335e75d30ff4376abe3b5f5ff22709)